src/router.js

```diff
@@ -17,8 +17,14 @@ export default function (options) {
       }
     },
     hooks: {
+      onUpdate: function (oldModel, newModel, data, actions) {
+        if(newModel.router && oldModel.router.path !== newModel.router.path &&
```

Comment:
I'm not sure if I follow.. surely at some point there will be a difference in `location.pathname` from `/one` to `/two`. At the moment this code triggers only when the location is actually changed. Not on page load like a subscription and not if you navigate from `/one` to `/one`